wee_slack.py

```diff
@@ -1326,14 +1328,14 @@ def store_message(self, message, team, from_me=False):
         if from_me:
             message.message_json["user"] = team.myidentifier
         self.messages[SlackTS(message.ts)] = message
-        if len(self.messages.keys()) > SCROLLBACK_SIZE:
-            mk = self.messages.keys()
-            mk.sort()
-            for k in mk[:SCROLLBACK_SIZE]:
-                msg_to_delete = self.messages[k]
-                if msg_to_delete.hash:
-                    del self.hashed_messages[msg_to_delete.hash]
-                del self.messages[k]
+
+        sorted_messages = sorted(self.messages.items())
```

Comment:
Isnt't this already sorted?

Reply:
I guess it depends on if `store_message` can be called out of order or not. `OrderedDict` sorts by insertion order, so if this is called with a ts that is less than any of the existing messages, it won't be sorted. I think I put `sorted` here to be sure of that.

Reply:
ok, go ahead then. :-)